### PR TITLE
reviewing acomplished task.

### PR DESCRIPTION
**Pull request by anabiz**
contribution and suggestions are welcome on how to improve the functionability of the solution.

### CODE REVIEW

File: src/brackets/index.js [hunk 1]
```diff
@@ -4,6 +4,25 @@
  * @param {string} str The string of brackets.
  * @returns {"valid" | "invalid"} Whether or not the string is valid.
  */
-function isValid(str) {}
+function isValid(str) {
+   if(str.length%2 > 0){
+        return "invalid";
+    }
```
Comment (bondz) [inline src/brackets/index.js:10]: Nice. Simplest check 👍

File: src/brackets/index.js [hunk 2]
```diff
@@ -4,6 +4,25 @@
...
+    var check ={ ")":"(", "]":"[", "}":"{" };
```
Comment (bondz) [inline src/brackets/index.js:11]: Why is this reversed?

File: src/roman-numerals/index.js [hunk 3]
```diff
@@ -4,6 +4,23 @@
  * @param {string} roman The all-caps Roman numeral between 1 and 3999 (inclusive).
  * @returns {number} The decimal equivalent.
  */
-function romanToDecimal(roman) {}
+function romanToDecimal(roman) {
+  
+  roman = roman.toUpperCase()
+    var rom = {"I":1, "V":5, "X":10, "L":50, "C":100, "D":500, "M":1000, "IV":4, "IX":9, "XL":40, "XC":90, "CD":400, "CM":900};
```
Comment (bondz) [inline src/roman-numerals/index.js:10]: Prefer `let` or `const` for variable declaration.

File: src/roman-numerals/index.js [hunk 4]
```diff
@@ -4,6 +4,23 @@
...
+    var bb = 0, bc = 0, ab ="";
```
Comment (bondz) [inline src/roman-numerals/index.js:11]: One line per declaration.

File: src/roman-numerals/index.js (hunk 3 above)
Comment (anabiz) [reply src/roman-numerals/index.js:10]: Noted.

File: src/roman-numerals/index.js (hunk 4 above)
Comment (anabiz) [reply src/roman-numerals/index.js:11]: Noted as well...
thanks

File: src/brackets/index.js (hunk 2 above)
Comment (anabiz) [reply src/brackets/index.js:11]: It can still work if reversed, but the code will be adjusted a bit. 
Thanks for the review...

**Pull request merged**